### PR TITLE
Reducing Rational 6KT error message rate

### LINKED ISSUE #950: The 6KT deprecation info is overly verbose
**Describe the bug**
When use an EV Azure Kinect sensor, the calibration format was 6KT, which is supported by the SDK. However, after sensor SDK 1.3, it will overly verbose printing a critical error: "Rational 6KT is deprecated (only supported early internal devices). Please replace your Azure Kinect with a retail device." It makes the old device / recordings almost unusable without disabling any error loggings. 

The logging can be found here: https://github.com/microsoft/Azure-Kinect-Sensor-SDK/blob/6762f44a13061fce5c239fecdaeb3726a53774ce/src/transformation/intrinsic_transformation.c#L37 

**To Reproduce**
Run sample code with an EV device. 

**Expected behavior**
Make it as a warning instead of critical error. And only print it out once. 

**Desktop (please complete the following information):**
 - OS with Version: Windows 1903
 - SDK Version:  1.3

### PR DESCRIPTION
**Pull request by wes-b**
### Description of the changes:
- Updates warning to only fire 1 time per session instead of many

### I tested changes on: 
- [x] Windows
- [ ] Linux

Manually tested to confirm change.

### CODE REVIEW

File: src/transformation/intrinsic_transformation.c [hunk 1]
```diff
@@ -248,8 +255,10 @@ static k4a_result_t transformation_unproject_internal(const k4a_calibration_came
         return K4A_RESULT_FAILED;
     }
 
-    if (camera_calibration->intrinsics.type == K4A_CALIBRATION_LENS_DISTORTION_MODEL_RATIONAL_6KT)
+    if (camera_calibration->intrinsics.type == K4A_CALIBRATION_LENS_DISTORTION_MODEL_RATIONAL_6KT &&
+        g_deprecated_6kt_message_fired == false)
     {
+        g_deprecated_6kt_message_fired = true;
         LOG_CRITICAL("Rational 6KT is deprecated (only supported early internal devices). Please replace your Azure "
```
Comment (yijiew) [inline src/transformation/intrinsic_transformation.c]: Do we want to log the information as Critical level? Maybe warning will be enough?

Comment (wes-b) [reply src/transformation/intrinsic_transformation.c]: The issue is still critical for customers, so it should stay critical. If the calibration turns up then a production device is needed.

**Comment by wes-b** [comment]
Thanks for the quick review

File: src/transformation/intrinsic_transformation.c (hunk 1 above)
Comment (yijiew) [reply src/transformation/intrinsic_transformation.c]: I see. It makes sense :)

**Pull request merged**